OpenRA.Mods.RA2/dependencies/readme.txt

```diff
@@ -0,0 +1,5 @@
+You should copy following files out of your main OpenRA install in here:
```

Comment:
If you want to go that way, you have to make sure that the assemblies from OpenRA/OpenRA become versioned.